test/e2e/lib/helpers/broker_event_transformation_test_with_source.go

```diff
@@ -263,7 +263,7 @@ func BrokerEventTransformationTestWithPubSubSourceHelper(client *lib.Client, aut
 	source := schemasv1.CloudPubSubEventSource(project, topicName)
 
 	// Create a target PubSub Job to receive the events.
-	lib.MakePubSubTargetJobOrDie(client, source, targetName, lib.E2EPubSubRespEventType)
+	lib.MakePubSubTargetJobOrDie(client, source, targetName, lib.E2EPubSubRespEventType /*empty schema*/, "")
```

Comment:
Why the schema is empty here? I thought we should put schemasv1.CloudPubSubEventDataSchema.

Reply:
because the CE here comes from the receiver Pod, which is not setting the schema... There is a transformation in the middle